fabric/externalLauncher/bin/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Calling them `externalLauncher` next to a dir `externalBuilder` seems to me a bit confusing.   

More importantly though, i think for clarity it would be good to keep stuff like these artifacts which are specific to the our particular use of fabric-sample network for our testing all as below a sample-network specific sub-dir.  Given where we currently keep some of that network stuff `utils/sample-network-testing` or alike would be one choice or it could also be a subdir of `integration`  or alike.  And as mentioned on the ercc PR, i think also stuff like ecc_mock is best moved there. Minimizes clutter on top-level filenamespace and also makes it easy to see what all fits together rather than having it spread out the whole file tree?

Reply:
well, right now it is not clear if externalLauncher will be only used for the test-network or eventually replace externalBuilder here. Since you also have suggested to try to keep `main.go` of ercc and ecc flexible to support both modes, I think we can also keep externalLauncher and externalBuilder together in the `fabric` folder. We can come back to this later, but I don't think this needs to be discussed now.